packages/device_info/example/android/app/src/main/AndroidManifest.xml

```diff
@@ -12,15 +12,6 @@
                   android:exported="true"
                   android:windowSoftInputMode="adjustResize">
         </activity>
-        <activity android:name=".MainActivity"
```

Comment:
I think this wouldn't work. You still need to declare an activity. It would just be a standard FlutterActivity instead of an activity in this package though.

Reply:
OK, I missed that one. Thank you.